### PR TITLE
volunteer search: restore functionality

### PR DESCRIPTION
**Pull request by breville**

### CODE REVIEW

File: pegasus/routes/form_routes.rb [hunk 1]
```diff
@@ -16,7 +16,13 @@
     content_type :json
     cache_control :private, :must_revalidate, max_age: 0
     form = insert_or_upsert_form(kind, params)
-    JSON.load(form[:data]).merge(secret: form[:secret]).to_json
+    data = JSON.load(form[:data]).merge(secret: form[:secret])
+    if form[:kind] == "VolunteerContact2015"
```
Comment (Erin007) [inline pegasus/routes/form_routes.rb:20]: Are we re-using the same form template from 2015? Anything about the year that needs updating?

File: pegasus/sites.v3/code.org/public/js/volunteer-local.js [hunk 2]
```diff
@@ -285,6 +286,45 @@ function contactVolunteer() {
...
+  var errors_count = errors.length;
+
+  for (var i = 0; i < errors_count; ++i) {
+    var error_id = '#volunteer-contact-' + errors[i].replace(/_/g, '-');
+    error_id = error_id.replace(/-[sb]s?$/, '');
+    $(error_id).parents('.form-group').addClass('has-error');
+  }
+
+  var error = '<font color="#a94442">An error occurred. All fields are required. Please check that all fields have been filled out properly.</font>';
+  $('#error-message').html(error).hide().fadeTo("normal", 1);
```
Comment (Erin007) [inline pegasus/sites.v3/code.org/public/js/volunteer-local.js:308]: just US/English volunteers or do we need a way to translate these error messages?

File: pegasus/routes/form_routes.rb (hunk 1 above)
Comment (breville) [reply pegasus/routes/form_routes.rb:20]: Apparently we dated this particular form but never bumped it, so it appears fixed at 2015 indefinitely.

File: pegasus/sites.v3/code.org/public/js/volunteer-local.js (hunk 2 above)
Comment (breville) [reply pegasus/sites.v3/code.org/public/js/volunteer-local.js:308]: We haven't localised any of the experience at /volunteer or /volunteer/local.

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/js/volunteer-local.js:308]: Actually /volunteer and /volunteer/local do have a lot localized already, but it's not a goal. Wouldn't worry about this for this PR.

File: pegasus/routes/form_routes.rb (hunk 1 above)
Comment (tanyaparker) [reply pegasus/routes/form_routes.rb:20]: Yep. Bad naming on my part. I had created this when we created the HocSignup form. While we bumped the HocSignup every year, we didn't bump the volunteer one.

**Pull request merged**